README.md

```diff
@@ -10,20 +10,23 @@ Welcome to **CyberBear**, a gamified platform designed to educate users on cyber
   - [Technology Stack](#technology-stack)
   - [Use Cases](#use-cases)
   - [Setup and Installation](#setup-and-installation)
-  - [Contribution Guidelines](#contribution-guidelines)
-    - [Development Requirements](#development-requirements)
+  - [Join Our Community](#join-our-community)
   - [License](#license)
 
 ## Project Overview
 CyberBear is part of **HackToFuture-2024**, under the Cyber Security domain (Problem Statement Code: CS-01). Our team aims to promote cybersecurity awareness by turning complex cybersecurity concepts into interactive games, utilizing a subscription model to unlock advanced features, and providing quizzes and real-time assistance with the help of a fine-tuned Large Language Model (LLM).
 
+![Gamified Learning](https://media.giphy.com/media/3o7abB06u9bNzA8lu8/giphy.gif)
+
```

Comment:
This gif doesn't seem to be related to this project?

Reply:
I'll remove this.